nixos/modules/services/x11/redshift.nix

```diff
@@ -32,53 +30,74 @@ in {
       description = ''
         Enable Redshift to change your screen's colour temperature depending on
         the time of day.
+
+        This module also supports Gammastep, look for
+        <literal>services.redshift.package</literal> and
+        <literal>services.redshift.executable</literal> options.
       '';
     };
 
-    temperature = {
-      day = mkOption {
-        type = types.int;
-        default = 5500;
-        description = ''
-          Colour temperature to use during the day, between
-          <literal>1000</literal> and <literal>25000</literal> K.
-        '';
-      };
-      night = mkOption {
-        type = types.int;
-        default = 3700;
-        description = ''
-          Colour temperature to use at night, between
-          <literal>1000</literal> and <literal>25000</literal> K.
-        '';
-      };
+    dawnTime = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "6:00-7:45";
+      description = ''
+        Set the time interval of dawn manually.
+
+        The times must be specified as HH:MM in 24-hour format.
+      '';
     };
 
-    brightness = {
-      day = mkOption {
-        type = types.str;
-        default = "1";
-        description = ''
-          Screen brightness to apply during the day,
-          between <literal>0.1</literal> and <literal>1.0</literal>.
-        '';
-      };
-      night = mkOption {
-        type = types.str;
-        default = "1";
-        description = ''
-          Screen brightness to apply during the night,
-          between <literal>0.1</literal> and <literal>1.0</literal>.
-        '';
-      };
+    duskTime = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "18:35-20:15";
+      description = ''
+        Set the time interval of dusk manually.
+
+        The times must be specified as HH:MM in 24-hour format.
+      '';
+    };
+
+    latitude = mkOption {
+      type = types.nullOr types.float;
+      default = null;
+      example = "0.0";
```

Comment:
I don't know if this is a bug, but setting it as a float was making it not appear in manual.